playground/src/components/module_response_view.tsx

```diff
@@ -12,14 +12,23 @@ export default function ModuleResponseView({
     return null;
   }
   let formattedData = response.data;
+  let formattedMeta = response.meta;
   try {
     formattedData = JSON.parse(formattedData);
   } catch (e) {
     // ignore
   }
+  try {
+    formattedMeta = JSON.parse(formattedMeta);
+  } catch (e) {
+    // ignore
+  }
```

Comment:
```suggestion
  try {
    formattedData = JSON.parse(formattedData);
  } catch (e) {
    // continue to show the non-formatted response string
  }
  try {
    formattedMeta = JSON.parse(formattedMeta);
  } catch (e) {
    // continue to show the non-formatted response string
  }
```

Improving my own old comment here, I believe.